website/docs/sdk-reference/community/laravel.md

```diff
@@ -249,3 +261,11 @@ Please see [CONTRIBUTING](https://github.com/Pod-Point/laravel-configcat/blob/ma
 ## License
 
 The MIT License (MIT). Please see [License File](https://github.com/Pod-Point/laravel-configcat/blob/main/LICENSE.md) for more information.
+
+---
```

Comment:
Hey there! Unfortunately, it is against our policy to put marketing backlinks to the docs. However, we have a partnership program where we are open to backlink exchanges or publishing guest blog posts.